src/Symfony/Installer/NewCommand.php

```diff
@@ -133,7 +139,22 @@ private function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
         // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
+        if (self::VERSION_LATEST === $this->version) {
+            return $this;
+        }
+
+        // Get the Symfony versions list (http://symfony.com/versions.json)
+        $versionsList = $this->getVersionsList();
```

Comment:
this is no loading the version list in cases where it is not needed (passing an exact version), which is wrong. The HTTP request should be done only when needed